app/helpers/parent_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module ParentHelper
+  def parent_promo_content(parent_item)
+    return nil if parent_item.blank?
+
+    {
+      url: document_path(parent_item['id'][1..-1]),
+      title: parent_item['title'].is_a?(Array) ? parent_item['title'].first : parent_item['title'],
```

Comment:
Coming from search results, title will always be an array, even if only having one value.